test/spec/BpmnPropertiesPanel.spec.js

```diff
@@ -314,7 +401,7 @@ describe('<BpmnPropertiesPanel>', function() {
       createBpmnPropertiesPanel({ container, eventBus });
 
       // then
-      expect(loadedSpy).to.have.been.calledOnce;
+      expect(loadedSpy).to.have.been.called;
```

Comment:
This basically tested internal API of the properties-panel before, cf. https://github.com/bpmn-io/properties-panel/pull/228